### DISCUSSION TITLE
Invoncify new branding

### DISCUSSION THREAD

**Discussion by AndresMorelos** [AndresMorelos/InvoncifyReleases#35] [Ideas]
This discussion was created in order to talk about the new branding for Invoncify app.

**Comment by sonalder-darlene** [comment]
Do you have any idea of what the logo of Invoncify should represents ?

**Reply by AndresMorelos** [reply]
I was thinking on keep it simple, it will need to represent a robust app, highly secured, but always letting the user know that it is an invoice app.

Do you have ideas about what the logo should represent?

**Reply by sonalder-darlene** [reply]
What about an envelope combine with a lock ? ✉️🔒

**Reply by sonalder-darlene** [reply]
![logo proposal](https://user-images.githubusercontent.com/73744170/159809870-18f3b839-1929-4d57-bd8c-e932864216ce.png)

**Reply by sonalder-darlene** [reply]
The font is open source by the way

**Reply by sonalder-darlene** [reply]
![logo-proposal-macOS-icon](https://user-images.githubusercontent.com/73744170/159812470-7b1b10e7-2490-47b9-af6b-b4db79bb4b8e.png)
I think green is a great color for Invoncify because :

- When numbers are green you're happier than when they're red 🤑
- It's common to have red, green and white colors to represent invoices and finances stuff 📈📊🧾
- For a security point of view it's seen as a safe color 🔐✅
- Popular ify apps (Spotify, Shopify) have green logos 🤷🏼‍♀️

The gradient was a bit randomly choosen.
Which color should I try ? Should it be one color or a gradient ? Do you like the idea ?

**Comment by AndresMorelos** [comment]
@sonalder-darlene hey men, sorry for the delay, I was focused at work, I was thinking recently and thought to keep an animal in the logo, I was looking for one and remember one of the more colorful of my country the Papagayo, here is an image about it.

![image](https://user-images.githubusercontent.com/6563162/160739061-9430ef78-630b-4971-9f07-eccbc2c091e9.png)

Your first logo was amazing, but what if we develop an logo with this animal?

**Reply by sonalder-darlene** [reply]
I'll try to do some versions. Do you like the font or nah ?

**Reply by AndresMorelos** [reply]
Yeah, the font is nice

**Comment by AndresMorelos** [comment]
New Branding initialize with the logo redesign https://github.com/AndresMorelos/Invoncify/discussions/85

next move: Landing Page

<img src="https://user-images.githubusercontent.com/6563162/173639943-716dae63-0f9a-4c4c-aade-271adf2edc8f.png" width="200" heigth="200"/>
